### DISCUSSION TITLE
How do I debug theme issues?

### DISCUSSION THREAD

**Discussion by simon-friedberger** [sharkdp/bat#2900] [Q&A]
The surrounding block and the line numbering is extremely faint in iterm2 with solarized dark enabled. Is there any way to find out which color it is using and why?

**Comment by keith-hall** [comment]
Themes are just xml files included as submodules from this repo, so they are quite easy to look at. This should be the line number color for example: https://github.com/braver/Solarized/blob/87e01090cf5fb821a234265b3138426ae84900e7/Solarized%20(dark).tmTheme#L21

**Reply by simon-friedberger** [reply]
Wait, where did you get which color bat is using for that part of the UI?

**Reply by keith-hall** [reply]
https://github.com/sharkdp/bat/blob/4790def1ef69134e825177b684ce903a111e7df8/src/printer.rs#L844

**Reply by simon-friedberger** [reply]
Oh, so bat is pulling in that theme definition and the color is basically a fixed color independent of the terminal theme?

**Reply by keith-hall** [reply]
I'm pretty sure that unless you're using an ANSI theme, your terminal theme is irrelevant :)

**Reply by simon-friedberger** [reply]
But it's still using the background color from the theme which is probably why I cannot see anything.
That's not a criticism, though. I am very happy that it's using the given background color. I change it to highlight active tmux panes.
